vendor/bat-native-ads/src/bat/ads/internal/tokens/refill_unblinded_tokens/request_signed_tokens_url_request_builder.cc

```diff
@@ -16,6 +16,10 @@
 #include "bat/ads/internal/security/security_util.h"
 #include "bat/ads/internal/server/ads_server_util.h"
 
+#if defined(RPILL_ENABLED)
```

Comment:
this is not correct, you need to use `#if BUILDFLAG(ENABLE_EXTENSIONS)` and include the buildflag header

Reply:
also please change to `ENABLE_RPILL` and same with the gn flag